acados/sim/sim_gnsf.c

```diff
@@ -0,0 +1,1757 @@
+/*
+ *    This file is part of acados.
+ *
+ *    acados is free software; you can redistribute it and/or
+ *    modify it under the terms of the GNU Lesser General Public
+ *    License as published by the Free Software Foundation; either
+ *    version 3 of the License, or (at your option) any later version.
+ *
+ *    acados is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ *    Lesser General Public License for more details.
+ *
+ *    You should have received a copy of the GNU Lesser General Public
+ *    License along with acados; if not, write to the Free Software Foundation,
+ *    Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *  
+ *    Author: Jonathan Frey
+ */
+
+// standard
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <assert.h>
+
+// acados
+#include "acados/utils/mem.h"
+#include "acados/utils/print.h"
+#include "acados/utils/timing.h"
+
+#include "acados/sim/sim_common.h"
+#include "acados/sim/sim_gnsf.h"
+#include "acados/sim/sim_irk_integrator.h"
+#include "acados/sim/sim_collocation_utils.h"
+
+// blasfeo
+#include "blasfeo/include/blasfeo_target.h"
+#include "blasfeo/include/blasfeo_common.h"
+#include "blasfeo/include/blasfeo_d_aux_ext_dep.h"
```

Comment:
BLASFEO _ext_dep should not be used anywhere in acados core... so either this (and the i_aux_ext_dep) header is useless, or the code has to be fixed to not use these routines